soh/soh/Enhancements/crowd-control/CrowdControl.cpp

```diff
@@ -162,8 +172,7 @@ void CrowdControl::ListenToServer() {
                 }
 
                 // check if effect can be executed
-                EffectResult result =
-                ExecuteEffect(incomingEffect->type.c_str(), incomingEffect->value, true);
+                EffectResult result = CrowdControl::CanApplyEffect(incomingEffect);
```

Comment:
it looks like the previous logic here was doing the equivalent of
```cpp
if (CrowdControl::CanApplyEffect(incomingEffect)) {
    result = GameInteractor::ApplyEffect(incomingEffect->giEffect);
}
```
now it seems to only check and never actually apply

if the `CanApplyEffect` check is moved into `ApplyEffect` as my earlier comment suggests, this line could just be
```cpp
EffectResult result = GameInteractor::ApplyEffect(incomingEffect->giEffect);
```